docs/resources/mysql_databases.md

```diff
@@ -6,54 +6,59 @@ subcategory: "MySQL"
 
 Provides a MySQL Database list resource.
 
-~> **NOTE:** This resource only supports VPC environment.
+~> **NOTE:** This resource only supports VPC environment. When you create an `ncloud_mysql` resource, a database is created by default with the value you enter for `database_name`. At least one database must always exist. The `ncloud_mysql_databases` resource is used if you need to create additional databases.
```

Comment:
(suggestion, not to use `you`) When a `ncloud_mysql` resource is created, a default database is automatically generated using the value specified in `database_name`. There is always one default database, and to create additional databases, you should use the `ncloud_mysql_databases` resource.